src/main/java/Main.java

```diff
@@ -32,21 +33,43 @@ public static void main(String[] args) {
 
         greet();
         String input = in.nextLine();
+        int inputTaskNumber;
+
         while (!input.equals("bye")) {
 
             String[] arrOfInputStrings = input.split(" ");
 
-            if (input.equals("list")) {
+            switch(arrOfInputStrings[0]) {
+            case "list":
                 displayList(tasks, taskCounter);
-            } else if (arrOfInputStrings[0].equals("mark")) {
-                int inputTaskNumber = Integer.parseInt(arrOfInputStrings[1]) - 1;
+                break;
+            case "todo":
+                tasks[taskCounter] = new Todo(input.substring(5));
+                taskCounter++;
+                break;
+            case "deadline":
+                String[] splitDeadlineDescription = input.substring(9).split(" /by ");
+                tasks[taskCounter] = new Deadline(splitDeadlineDescription[0], splitDeadlineDescription[1]);
+                taskCounter++;
+                break;
+            case "event":
+                String[] splitEventDescription = input.substring(6).split(" /at ");
+                tasks[taskCounter] = new Event(splitEventDescription[0], splitEventDescription[1]);
+                taskCounter++;
+                break;
+            case "mark":
+                inputTaskNumber = Integer.parseInt(arrOfInputStrings[1]) - 1;
                 tasks[inputTaskNumber].markAsDone();
-            } else if (arrOfInputStrings[0].equals("unmark")) {
-                int inputTaskNumber = Integer.parseInt(arrOfInputStrings[1]) - 1;
+                break;
+            case "unmark":
+                inputTaskNumber = Integer.parseInt(arrOfInputStrings[1]) - 1;
                 tasks[inputTaskNumber].markAsUndone();
-            } else {
-                tasks[taskCounter] = new Task(input);
-                taskCounter++;
+                break;
+            default:
+                System.out.println("-----------------------------------------------------");
+                System.out.println("Invalid input\nPlease use the format: [type_of_task] [task_description]");
```

Comment:
Maybe consider dividing this line into two

Reply:
Are you referring to line 70 being too long? The line has 109 characters so I thought it was still fine (since the "soft cap" is at 110).